addons/hr/hr.py

```diff
@@ -272,7 +272,7 @@ def _message_get_auto_subscribe_fields(self, cr, uid, updated_fields, auto_follo
 
 class hr_department(osv.osv):
     _name = "hr.department"
-    _description = "HR Department"
+    _description = "Department"
```

Comment:
@atp-odoo this is hacking the create to avoid sending a message and then sending it yourself. Why do you want to do that?